# DP: the wininst-* files cannot be built within Debian, needing a
# DP: zlib mingw build, which the zlib maintainer isn't going to provide.

--- a/Lib/distutils/command/bdist_wininst.py
+++ b/Lib/distutils/command/bdist_wininst.py
@@ -360,7 +360,10 @@
             sfix = ''
 
         filename = os.path.join(directory, "wininst-%.1f%s.exe" % (bv, sfix))
-        f = open(filename, "rb")
+        try:
+            f = open(filename, "rb")
+        except IOError, msg:
+            raise DistutilsFileError, str(msg) + ', %s not included in the Debian packages.' % filename
         try:
             return f.read()
         finally:
